README.md

```diff
@@ -200,24 +193,22 @@ I usually perform a second checkout of the _gh-pages_ branch and copy the html f
     rsync -a docs/html/ ../client-py-web/
 
 
-PyPi Publishing (notes for SMART team)
---------------------------------------
+## PyPi Publishing (notes for SMART team)
 
-Using setuptools (*Note*: Alternatively, you can use twine https://pypi.python.org/pypi/twine/):
+Using flit (*Note*: Alternatively, you can use [twine](https://twine.readthedocs.io/)):
 
 ### Make sure that you have the PyPi account credentials in your account
 
     copy server.smarthealthit.org:/home/fhir/.pypirc to ~/.pypirc
 
 ### Test the build
 
-    python setup.py sdist
-    python setup.py bdist_wheel
+    python3 -m build
```

Comment:
I tested the resulting sdist & and bdist from the new approach vs the old tarballs - new ones are still good.